Framework/PDFExporter/PDFExporter/Categories/UIKit/UIView+PDFExporterStatePersistance.m

```diff
@@ -0,0 +1,86 @@
+//
+//  UIView+StatePersistance.m
+//  PDFExporter
+//
+//  Copyright © 2016 3Pillar Global. All rights reserved.
+//
+
+#import "UIView+PDFExporterStatePersistance.h"
+#import <objc/runtime.h>
+
+static void * const kUIViewPersistenceStatesAssociatedStorageKey = (void *)&kUIViewPersistenceStatesAssociatedStorageKey;
+static void * const kUIViewPersistStateAssociatedStorageKey = (void *)&kUIViewPersistStateAssociatedStorageKey;
+
+@interface UIView (PDFExporterStatePersistancePrivate)
+
+@property (nonatomic) NSMutableArray<UIViewPersistenceState *> *persistenceStates;
+
+@end
+
+@implementation UIView (PDFExporterStatePersistance)
+
+- (NSMutableArray<UIViewPersistenceState *> *)persistenceStates {
```

Comment:
:+1: For using generics...it would be nice to use them everywhere as well.  Makes Swift use a lot easier